scripts/increment-cargo-version.sh

```diff
@@ -146,6 +146,18 @@ done
 # Update cargo lock files
 scripts/cargo-for-all-lock-files.sh tree >/dev/null
 
+# Only apply related changes
+(
+  shopt -s globstar
+  # shellcheck disable=SC2035
+  git diff --unified=0 **/Cargo.lock >cargo-lock-patch
```

Comment:
I think we can remove the shellcheck exceptions by prefacing the globstar with `./`. I tried it locally on v1.18 and the output was identical.

```suggestion
    git diff --unified=0 ./**/Cargo.lock >cargo-lock-patch
```